packages/assets-controllers/src/NftDetectionController.ts

```diff
@@ -154,6 +149,231 @@ export interface NftDetectionConfig extends BaseConfig {
+  malicious_score: string;
+  attack_types: object;
+};
+
+export type Market = {
+  floorAsk?: FloorAsk;
+  topBid?: TopBid;
+};
+
+export type TokenResponse = {
+  chainId: number;
+  contract: string;
+  tokenId: string;
+  /** @description Can be erc721, erc115, etc. */
+  kind?: string;
+  name?: string;
+  image?: string;
+  imageSmall?: string;
+  imageLarge?: string;
+  metadata?: Metadata;
+  description?: string;
+  /** @description Can be higher than one if erc1155. */
+  supply?: number;
+  remainingSupply?: number;
+  /** @description No rarity for collections over 100k */
+  rarityScore?: number;
+  rarity?: number;
+  /** @description No rarity rank for collections over 100k */
+  rarityRank?: number;
+  media?: string;
+  /** @default false */
+  isFlagged?: boolean;
+  /** @default false */
+  isSpam?: boolean;
+  /** @default false */
+  isNsfw?: boolean;
+  /** @default false */
+  metadataDisabled?: boolean;
+  lastFlagUpdate?: string;
+  lastFlagChange?: string;
+  collection?: Collection;
+  lastSale?: LastSale;
+  topBid?: TopBid;
+  /** @description The value of the last sale.Can be null. */
+  lastAppraisalValue?: number;
+  attributes?: Attributes[];
+};
+
+export type TopBid = {
+  id?: string;
+  price?: Price;
+  source?: {
+    id?: string;
+    domain?: string;
+    name?: string;
+    icon?: string;
+    url?: string;
+  };
+};
+
+export type LastSale = {
+  /** @description Deprecated. Use `saleId` instead. */
+  id?: string;
+  /** @description Unique identifier made from txn hash, price, etc. */
+  saleId?: string;
+  token?: {
+    contract?: string;
+    tokenId?: string;
+    name?: string;
+    image?: string;
+    collection?: {
+      id?: string;
+      name?: string;
+    };
+  };
+  orderSource?: string;
+  /**
+   * @description Can be `ask` or `bid`.
+   * @enum {string}
+   */
+  orderSide?: 'ask' | 'bid';
+  orderKind?: string;
+  orderId?: string;
+  from?: string;
+  to?: string;
+  amount?: string;
+  fillSource?: string;
+  block?: number;
+  txHash?: string;
+  logIndex?: number;
+  batchIndex?: number;
+  /** @description Time added on the blockchain */
+  timestamp?: number;
+  price?: Price;
+  washTradingScore?: number;
+  royaltyFeeBps?: number;
+  marketplaceFeeBps?: number;
+  paidFullRoyalty?: boolean;
+  feeBreakdown?: FeeBreakdown[];
+  isDeleted?: boolean;
+  /** @description Time when added to indexer */
+  createdAt?: string;
+  /** @description Time when updated in indexer */
+  updatedAt?: string;
+};
+
+export type FeeBreakdown = {
+  kind?: string;
+  bps?: number;
+  recipient?: string;
+  source?: string;
+  rawAmount?: string;
+};
+
+export type Attributes = {
+  /** @description Case sensitive */
+  key?: string;
+  /** @description Can be `string`, `number, `date, or `range`. */
+  kind?: string;
+  /** @description Case sensitive. */
+  value: string;
+  tokenCount?: number;
+  onSaleCount?: number;
+  floorAskPrice?: Price;
+  /** @description Can be null. */
```

Comment:
Similar, why is this not reflected in the type?

Reply:
Definitely, the floorAskPrice and topBidValue can be null
example
```
          {
            "key": "Speed",
            "kind": "number",
            "value": "90",
            "tokenCount": 1,
            "onSaleCount": 0,
            "floorAskPrice": null,
            "topBidValue": null,
            "createdAt": "2023-08-25T03:53:04.916Z"
          }
```
I can add 
```
  floorAskPrice?: Price | null;
  topBidValue?: number | null;
```